### ISSUE TITLE
Implement Varnish in the plone.org stack

### ISSUE THREAD

**Issue by ericof** [plone/plone.org#51]

**Comment by rioksane** [comment]
@jensens @ericof can we close this issue, since the site is already in production? Just cleaning up the list here.

**Comment by ericof** [comment]
Keep it here. I will review that later as well

**Comment by jensens** [comment]
We still need to load a useful profile for p.a.caching, then its done.

**Comment by mauritsvanrees** [comment]
Whether Varnish is installed or not, we should probably add `plone.app.caching` to the packages and install it.  The control panel is not there, I looked in Classic UI.

Ah, in `portal_setup` I _do_ see the `plone.app.caching:default` profile.  So the package is available in the backend, but the profile is hidden from the Add-ons control panel.  We could apply the default profile via portal_setup, and then in the control panel apply the without-caching-proxy profile as a start.  Definitely something for the test site first.

Or is this considered not necessary because we use Volto and all requests go through the restapi anyway?

I only looked at this because I wanted to compare some plone.org settings with those on another site. :-)

**Comment by davisagli** [comment]
@mauritsvanrees I agree we need plone.app.caching here. It can be a problem to serve a site through a CDN without proper cache headers. And we need them also for the REST API.
